tests/test_authentication.py

```diff
@@ -115,7 +115,7 @@ def test_login_required_roles_redirects_to_error_if_missing_roles(
         assert mock_request.status_code == 302
         assert (
             mock_request.location
-            == "https://authenticator/service/user?roles_required=COF_ADMIN|COF_TEST"
+            == "https://authenticator/service/user?roles_required=COF_ADMIN%7CCOF_TEST"
```

Comment:
hope this is a correct change ?

Reply:
Yes, the test was failing with (pipe)|, replacing it with %7C fixes that.

Reply:
After having a chat with @hamzabinkhalid it was a valid change